File: backend/src/main/java/capstone/rt04/retailbackend/request/refund/UpdateRefundLineItemHandlerRequest.java
package capstone.rt04.retailbackend.request.refund;

import capstone.rt04.retailbackend.util.enums.RefundProgressEnum;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class UpdateRefundLineItemHandlerRequest {
    private Long refundLineItemId;
    private String refundProgressEnum;
    private Long staffId;
    private Integer quantityConfirmedRefunded;
}

File: frontend/staff-web-app/src/models/refund/UpdateRefundLineItemHandlerRequest.js
class UpdateRefundLineItemHandlerRequest {
  refundLineItemId;
  refundProgressEnum;
  staffId;
  quantityConfirmedRefunded

  constructor(refundLineItemId, refundProgressEnum, staffId, quantityConfirmedRefunded) {
    this.refundLineItemId = refundLineItemId;
    this.refundProgressEnum = refundProgressEnum;
    this.staffId = staffId;
    this.quantityConfirmedRefunded = quantityConfirmedRefunded;
  }
}

export default UpdateRefundLineItemHandlerRequest;
